include/zrepl_mgmt.h

```diff
@@ -158,6 +158,11 @@ typedef struct zvol_info_s {
 
 	/* Will be used to singal ack-sender to exit */
 	uint8_t		conn_closed;
+
+	/* Rebuild flags to quiesce IOs */
+	uint8_t		quiesce_requested;
```

Comment:
can we use these two as bits in 'flags' variable?

Reply:
and create setter/getter macros around these variables

Reply:
ok.. we can't set them as bits, as there is no lock in reading these variables

Reply:
ok